app/src/main/java/com/github/se/wanderpals/ui/screens/trip/map/Map.kt

```diff
@@ -311,8 +311,8 @@ fun Map(
                 icon = BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_AZURE),
                 onInfoWindowClick = {
                   // check if stop.stopId contains a "/" and if it does, split it and get the first
```

Comment:
need to update the comment to match the change of the uid of a stop

Reply:
Thank you for the catch !